<launch>
  <group ns="soft_drone">
    <node pkg="softdrone_target_pose_estimator"
          name="plotter_node"
          type="plotter_node"
          output="screen">
    </node>

    <rosparam command="load" file="$(find softdrone_target_pose_estimator)/config/plotter_params.yaml" subst_value="true"/>
  </group>
</launch>
